### DISCUSSION TITLE
Slightly different rules import process from SQ one

### DISCUSSION THREAD

**Discussion by natixis-caen** [green-code-initiative/creedengo-csharp-sonarqube#5] [General]
Hi,

Thanks for your work great work! 

In the readme, I saw that you use a "slightly different rules import process" from the generated plugin. What is the purpose of this custom process? 
Did you create this project from scratch, from another one or by being inspired from the C# code in https://github.com/SonarSource/sonarqube-roslyn-sdk ?

Thanks

Ludo

**Comment by vdebellabre** [comment]
The process uses [Sonar Scanner for .Net](https://docs.sonarsource.com/sonarqube/latest/analyzing-source-code/scanners/sonarscanner-for-dotnet/) to import ecoCode rule analysis in to Sonar. However this tool is based on a project build output log, which doesn't contain important information like description or category, this is why we have a custom part in the import process
